finance/funding_requests.md

```diff
@@ -0,0 +1,81 @@
+
+
+The Astropy project wants to make life easy for all the volunteers and people working towards our shared goal; on the other hand, we also need to make sure we spend grant money in a fiscally responsible way and as allowed by donors or funding agencies. This policy thus aims to strike a balance between as little bureaucracy as possible while also ensuring that money is spend responsibly and that Astropy has all the documentation that we required for any of the grants given to Astropy.
+
+
+
+Funding requests can be made to the Interim Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+If you have questions on how to apply the rules, please email finance@astropy.org .
+
+
+## Travel
+
+Travel requests should be made as early as possible and at least 1 month before travel.   Requests made less than 1 month before the travel is to occur will still be considered but should include a justification for the timing.  
+
+1. Send an email to finance@astropy.org with the following information:
+* Name
+* Description
+* Total Request
+* Itemized Budget
+* Advance needed
+
+Costs for per diem can use the [US Government per diem rates | https://www.gsa.gov/travel/plan-book/per-diem-rates].  These should be considered maximum values for the trips and request less than that amount can be submitted for reimbursement.
+
+Flights should be booked in economy class.   If the flight is over 4 hours
```

Comment:
"should" might be interpreted as "must" by some readers, but this suggestion I think clears that up a bit
```suggestion
In general, flights should be booked in economy class.   If the flight is over 4 hours
```